src/frontend/parquet/parquet.jl

```diff
@@ -202,12 +208,22 @@ end
 Base.:(==)(a::DiagPara, b::DiagPara) = Base.isequal(a, b)
 
 include("common.jl")
-
 include("operation.jl")
-
 include("filter.jl")
-include("vertex4.jl")
+include("to_graph.jl")
 
+const vertex4I_diags = Dict{Int,Vector{Graph}}()
+function initialize_vertex4I_diags(; filter=[NoHartree], spinPolarPara::Float64=0.0)
```

Comment:
Can we add a docstring? Why is this function necessary / why only partitions (3, 0, 0) and (4, 0, 0)?

Reply:
This function initializes the fully-irreducible diagrams from GV, which will be used in Parquet.vertex4.